app/upload/views.py

```diff
@@ -200,9 +203,14 @@ def delete(r_id_type, r_id, filecode):
     return jsonify(response), 200
 
 
-@upload.route('/<request_id>', methods=['GET'])
-def get(request_id):
-    return jsonify({}), 200
+@upload.route('/<response_id>', methods=['GET'])
```

Comment:
We could need it eventually (for the API for Agencies). But it can be removed at this point, since that will required authentication workflow.